software/erxes.yml

```
@@ -5,6 +5,7 @@ licenses:
   - GPL-3.0
```

```suggestion
  - AGPL-3.0
  - ⊘ Proprietary
```

It seems the license has changed since this was first submitted:

> https://github.com/erxes/erxes/blob/master/LICENSE.md
> erxes is not permitted to be hosted as a SaaS version to compete with erxes Inc.

The project's description itself advertises it as "source available":

> Source available experience management infrastructure

You can replace `⊘ Proprietary` with `Commons-Clause` if you like, since this is very close to the Commons Clause (you cannot sell the software)

---

Commons clause seems reasonable to me